cloudpickle/__init__.py

```diff
@@ -4,4 +4,6 @@
 from cloudpickle.cloudpickle import *  # noqa
 from cloudpickle.cloudpickle_fast import CloudPickler, dumps, dump  # noqa
 
+Pickler = CloudPickler
```

Comment:
```suggestion
# Conform to the convention used by serialization libraries, which expose their
# Pickler subclass at top-level under the  "Pickler" name.
Pickler = CloudPickler
```

Reply:
thank you! Sorry, that you have to re-write the entire PR ;)

Reply:
No worries, thank your for taking the time to submit a patch. I'm simply maximizing information exposure as many parts of `cloudpickle`'s code can look obscure at first sight.